### ISSUE TITLE
Suggestion: Add custom username colors by role.

### ISSUE THREAD

**Issue by SaveAmerica** [RocketChat/feature-requests#887]
It would be great to have a built in feature like Discord or Guilded have where you can designate a color for the usernames of a specific role to differentiate users in the chat. 

If anyone knows a workaround for how to do this through custom CSS, please let me know.

**Comment by jsrjhbaryan959** [comment]
Yes, I know custom CSS, I would like to work on this, Please assign this to me.

**Comment by SaveAmerica** [comment]
> Yes, I know custom CSS, I would like to work on this, Please assign this to me.

If you are able to do this, feel free to go for it.

**Comment by SaveAmerica** [comment]
It looks like I don't have the ability to assign the issue to anybody.

**Comment by lakinweaver** [comment]
This used to be possible using CSS by targeting the role tags, and I would love to see this re-implemented. For example:

`[title="Example Role"],[data-role="Example Role"] {color:white !important;background:#ffb366 !important;}`

This CSS would change the text and background color of all "Example Role" user tags.

Beginning with (I'm pretty sure) the 5.0 release, role tags no longer seem to have a "data-role" HTML attribute. You can still target all role tags at the same time with CSS, but I have not found a way to target individual role tags...

Any new info on this would be greatly appreciated. Maybe I'm missing something!
